src/plugins/plugin.tooltip.js

```diff
@@ -1076,6 +1076,7 @@ export default {
 
 	afterDraw(chart) {
 		const tooltip = chart.tooltip;
+
```

Comment:
Are we triggering the events for custom tooltip or should we bail out earlier?

Reply:
I was unsure about that and my initial thought was to bail out and not do anything. I wasn't sure if anyone was using those events to do anything.